features/support/ui/pages/money_manager_circumstances_changed.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManagerCircumstancesChanged < UI::Page
+    set_url '/en/tools/money-manager/circumstances-changed'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
+    element :received_first_payment_false,
+            '#questionnaire_received_first_payment_false'
+    element :single_or_in_couple_single,
+            '#questionnaire_single_or_in_couple_single'
+    element :single_or_in_couple_couple,
+            '#questionnaire_single_or_in_couple_couple'
```

Comment:
Yes, I like this. The [new version](https://github.com/moneyadviceservice/frontend/pull/1764/files#diff-4b4d49b1134c3236d7ddad8e9712f677) is more verbose since we need to include the value (both radio buttons have the same `name`), but clearer and more explicit, I think.

The PR in `universal_credit` is approved, but I'll hold off on merging it until this is also approved - so let me know if you think this is OK now!